### ISSUE TITLE
Modifier Key Passthrough

### ISSUE THREAD

**Issue by mikerm19** [paidem/guacozy#24]
I noticed some behaviors I have questions on:

CTRL and ALT don't seem to pass through at all, is that possible? CTRL+C and CTRL+V on Windows doesn't seem to work (it just passes the C character, etc).

The Shift key appears to have an odd behavior. If you hold it down for a couple seconds it invokes the Sticky Keys dialog on the remote host (but not on the local host if Windows). Maybe I'm just too slow. ;)

**Comment by paidem** [comment]
CTRL and ALT pass through normally, you can see it if you enable OnScreen Keyboard on remote server. It displays which key it understands as pressed. But yes, I can see that although Windows can see key as pressed, it doesn't work and prints characters instead. I will test in with original Guacamole and search for similar problems for it. 

Clipboard sync doesn't work on Windows (between remote host and browser) anyway for some other reasons (it doesn't work if you select text and try to use copy/paste). I didn't find a solution yet, but I will open separate issue.

On sticky keys - it is a standard windows behavior, maybe you have disabled it on your local system. Press shift for too long - windows seems that you are retarded and tries to help.

**Comment by paidem** [comment]
BTW, thanks for using Guacozy. It is a very young project and using it / finding bugs helps a lot!

**Comment by mikerm19** [comment]
Thanks for writing it! It's great.

I've been trying to learn more about Docker so I can get some logs out. I can't SSH into my Dell switches, even though SSH to the Ubuntu servers works, so I will have some more for ya. :)

**Comment by paidem** [comment]
I tried searching same issue for Guacamole and found that Ctrl doesn't work if you select "Failsafe" keyboard layout, and it is the defaul in Guacozy. I switched to other one (English US keyboard) and Ctrl works!. 
https://sourceforge.net/p/guacamole/discussion/1110834/thread/0a56edb4/#3dd1

I will make "English US keyboard" default and put a notice in admin.  Will be released soon'ish in 0.0.27. For now - just change layout in your settings.
